src/drivers/QtTestDriver.cpp

```diff
@@ -0,0 +1,55 @@
+#include <cucumber-cpp/internal/drivers/QtTestDriver.hpp>
+
+#include <QtTest/QtTest>
+#include <QString>
+#include <QTextStream>
+#include <QTemporaryFile>
+
+#include <QObject>
+
+namespace cucumber {
+namespace internal {
+
+class QtTestObject: public QObject
+{
+    Q_OBJECT
+
+public:
+    QtTestObject(QtTestStep* step)
+    { Step = step; }
+    virtual ~QtTestObject() {}
+
+private:
+    QtTestStep* Step;
+
+private slots:
+    void test()
+    { Step->body(); }
+};
+
+
+const InvokeResult QtTestStep::invokeStepBody() {
+
+    QTemporaryFile file;
+    QString fileName;
+    if (file.open()) {
+        fileName = file.fileName();
+    }
+    file.close();
+
+    QtTestObject test(this);
+    int returnValue = QTest::qExec(&test, QStringList() << "test" << "-o" << fileName);
```

Comment:
I have to say I'm not thrilled about writing a file for each scenario step. Isn't there any other way? Perhaps diving deeper into the implementation? I see QTest::qExec doing plenty of things that are not needed by a step run.